db/index.ts

```diff
@@ -28,23 +29,187 @@ CREATE TABLE IF NOT EXISTS settings (
 );
 `;
 
-export const expo = SQLite.openDatabaseSync('period.db');
+let expo: SQLite.SQLiteDatabase | null = null;
+let db: ReturnType<typeof drizzle> | null = null;
+let initializationPromise: Promise<void> | null = null;
 
-// Execute the migration
-expo.execSync(MIGRATION_TABLES);
+export async function deleteDatabaseFile(): Promise<void> {
```

Comment:
Given `initializationPromise` and `expo`, I have little confidence that this function is doing what it's supposed to do and will actually remove the DB reliably. Such a critical routine shouldn't be obfuscated - it should be straightforward so people can analyse it.